test/built-ins/Temporal/PlainDateTime/prototype/since/roundingmode-trunc-is-default.js

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.plaindatetime.prototype.since
+description: Truncation (trunc) is the default rounding mode
+includes: [temporalHelpers.js]
+features: [Temporal]
+---*/
+
+const earlier = new Temporal.PlainDateTime(2019, 1, 8, 8, 22, 36, 123, 456, 789);
+const later = new Temporal.PlainDateTime(2021, 9, 7, 12, 39, 40, 987, 654, 321);
+
+TemporalHelpers.assertDuration(
+  later.since(earlier, { smallestUnit: "minutes" }),
+  0, 0, 0, 973, 4, 17, 0, 0, 0, 0,
+  "trunc is the default (1)"
```

Comment:
I'd recommend reflecting in the assertion messages why we have two of these tests — one would round down, and one would round up.

Reply:
Sounds good -- done.